src/app.js

```diff
@@ -2,26 +2,80 @@
 //We've got some basic info about Karen's home
 //Debug the type of data provided
 //Return the types concatenated in a single variable
+function moreAboutHome(address,distanceFromTown, hasNeighbours){
+    let typeOfAddress = typeof address;
+    let typeOfDistance = typeof distanceFromTown;
+    let typeOfNeighbours = typeof hasNeighbours;
+
+    return (typeOfAddress+typeOfDistance+typeOfNeighbours);
+
+}
 
 //Progression 2:
 //Check if the data given is of the right type
 //parents = String, noOfSiblings = Number, isNuclearFamily = Boolean
+function moreAboutKaren(parents, noOfSiblings, isNuclearFamily){
+    let isParents=typeof parents === "string";
```

Comment:
try to reduce the number of lines. Is it necessary to write 3 variables to concadinate with each other